callouts/java/service-callout/src/main/java/service/ServiceCallout.java

```diff
@@ -66,10 +66,11 @@ public class ServiceCallout {
     private String certKeyPath;
     private int serverThreadCount;
     private boolean enablePlainTextPort;
+    private boolean enableTls;
 
     protected ServiceCallout(Builder<?> builder) {
         this.ip = Optional.ofNullable(builder.ip).orElse("0.0.0.0");
-        this.port = Optional.ofNullable(builder.port).orElse(443);
+        this.port = Optional.ofNullable(builder.port).orElse(443); // Only used if enableTls is true
```

Comment:
If this `port` field is only used for TLS, would be clearer to name it `securePort`.